ground/src/main/java/com/google/android/ground/ui/common/AbstractMapFragmentWithControls.kt

```diff
@@ -55,7 +56,7 @@ abstract class AbstractMapFragmentWithControls : AbstractMapContainerFragment()
     viewLifecycleOwner.lifecycleScope.launch {
       repeatOnLifecycle(Lifecycle.State.STARTED) {
         getMapViewModel().location.collect {
-          val taskData = it?.toCaptureLocationResult()
+          val taskData = it?.toCaptureLocationResult(timestampMillis = Date().time)
```

Comment:
Why do we save the current time in two places, here and in the `CaptureLocationTaskMapFragment`? 

@kenstershiro My initial intuition was that we should store timestamp when the user clicks "Capture Location" button, not when the OS gives a location update, which would usually be slightly sooner. The degenerate case would be if the OS stops giving location updates (no GPS signal, location services disabled) - in that case, would we want to capture the time "Capture location" was clicked, or the time the last location was known? Can you ask @lecrabe and @jo-spek?

Reply:
We wanted the formatted lat/lng for the info card. Removed the dependency of `CaptureLocationTaskData` from this class and directly created the formatted location text.

Reply:
Discussed with @jo-spek

His main concern is that no other task stores a timestamp. Putting this PR on hold until the discussion is finalized.